PxGraf/Models/Metadata/PxGrafMetaExtensions.cs

```diff
@@ -0,0 +1,116 @@
+﻿using Px.Utils.Language;
+using Px.Utils.Models.Metadata;
+using Px.Utils.Models.Metadata.Dimensions;
+using Px.Utils.Models.Metadata.Enums;
+using Px.Utils.Models.Metadata.MetaProperties;
+using PxGraf.Data;
+using PxGraf.Data.MetaData;
+using PxGraf.Settings;
+using PxGraf.Utility;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+
+namespace PxGraf.Models.Metadata
+{
+    /// <summary>
+    /// Extension methods for <see cref="CubeMeta"/> objects.
+    /// </summary>
+    public static class PxGrafMetaExtensions
+    {
+        public static MatrixMetadata ToMatrixMetadata(this CubeMeta pxGrafMeta)
+        {
+            List<Dimension> dimensions = new(pxGrafMeta.Variables.Count);
+            Dictionary<string, MetaProperty> matrixProperties = [];
+            if (pxGrafMeta.Note != null)
+            {
+                Dictionary<string, string> note = [];
+                foreach (var lang in pxGrafMeta.Note.Languages)
+                {
+                    note[lang] = $"\"{pxGrafMeta.Note[lang]}\""; // Meta property values must be enclosed
```

Comment:
Why are these quotation marks added here?

Reply:
This was required earlier when Px.Utils provided metadata strings with quotation marks, but this was changed later. Fixing it here.